pyproject.toml

```diff
@@ -8,8 +8,6 @@ description = "Specification, solution, and simulation of finite-horizon dynamic
 requires-python = ">=3.12"
 dependencies = [
     "dags>=0.4.1",
-    "jax>=0.7",
```

Comment:
I don't think we can remove these here, since this is the list of the library dependencies.